src/res/shop.yaml

```diff
@@ -106,3 +106,10 @@ items:
     price: 120
     image: default.svg
     available: true
+
+  - title: Flutter Book (French)
+    description: |
+      Développez vos applications mobiles multiplateformes avec Dart (470 pages)
+    price: 100
```

Comment:
I have the feeling 100 Yotas is a bit too much, it corresponds to 20 small PRs which looking at the trafic on our GitHub profile seems to be a lot. How about setting it to 50 Yotas ?

Reply:
@elhmn done. You're right